packages/vike-vue/src/components/Config/Config-client.ts

```diff
@@ -0,0 +1,23 @@
+export { Config }
+
+// Same as ./Config-server.ts but importing useConfig-client.js
+
+import { defineComponent, PropType } from 'vue'
+import { useConfig } from '../../hooks/useConfig/useConfig-client.js'
+import type { ConfigFromHook } from '../../+config.js'
+
+const Config = defineComponent({
+  name: 'Config',
+  props: {
+    head: Object as PropType<ConfigFromHook['Head']>,
+    title: String,
+    description: String,
+    image: String,
+  },
+  inheritAttrs: false,
+  setup: (props) => () => {
+    const config = useConfig()
+    config({ Head: () => props.head, ...props })
```

Comment:
I’ve updated the code. It might not include everything from your review, but what do you think?

Note that I didn't put watchers (yet), since `pageContext._headAlreadySet` always undefined.
see https://github.com/vikejs/vike-vue/blob/b65d7866736bf52249a5fd6918b65de2ef4db6db/packages/vike-vue/src/hooks/useConfig/useConfig-client.ts#L24

Reply:
> Also here you process Head on client-side

Now it doesn't